apollo-normalized-cache-sqlite/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/sql/SqlNormalizedCache.kt

```diff
@@ -27,6 +30,19 @@ class SqlNormalizedCache internal constructor(
     return nextCache?.loadRecord(key, cacheHeaders)
   }
 
+  override fun stream(key: String, cacheHeaders: CacheHeaders): JsonReader? {
+    return try {
+      cacheQueries.recordForKey(key)
+          .executeAsList()
+          .firstOrNull()
+          ?.let {
+            BufferedSourceJsonReader(Buffer().writeUtf8(it.record))
```

Comment:
I'm not sure if this involves copying bytes or if the `Buffer` will automagically do the right thing. It might be worth investigating this.